ops/services/alerts/app_service_metrics/main.tf

```diff
@@ -480,3 +480,38 @@ and duration >= 180000
     ]
   }
 }
+
+resource "azurerm_monitor_scheduled_query_rules_alert" "frontend_fail_to_communicate_with_backend_alert" {
+  name                = "${var.env}_frontend_fail_to_communicate_with_backend_alert"
+  description         = "Action will be triggered when API request to feature flag endpoint failures > 0"
+  location            = data.azurerm_resource_group.app.location
+  resource_group_name = var.rg_name
+  severity            = var.severity
+  frequency           = 60
+  time_window         = 60
```

Comment:
⚠️ set this to 60 minutes as well to try and maximize the chances someone has hit the site and run into this error during this time period (again, please let me know if I should adjust 😅)
[see time window](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/monitor_scheduled_query_rules_alert#time_window)